synapse/rest/admin/__init__.py

```diff
@@ -238,6 +238,10 @@ def register_servlets(hs: "HomeServer", http_server: HttpServer) -> None:
     """
     Register all the admin servlets.
     """
+    # Admin servlets aren't registered on workers.
+    if hs.config.worker.worker_app is not None:
+        return
```

Comment:
I think a few read-only servlets (ListRoom,Version) would probably be safe to run on workers... but we probably don't take advantage of that?

(Just wondering if there's a user-facing change that we need to call out here)

Reply:
> We do not register any of these for workers right now...except in unit tests.

Super, thanks for spelling it out for me.